apps/webapp/app/platform/zodWorker.server.ts

```diff
@@ -443,7 +573,8 @@ export class ZodWorker<TMessageCatalog extends MessageCatalogSchema> {
 
     await this.enqueue(helpers.job.task_identifier, payload, {
       runAt: helpers.job.run_at,
-      queueName: helpers.job.queue_name ?? undefined,
+      // TODO: fix queueName, maybe use reschedule helper
+      // queueName: helpers.job.job_queue_id ?? undefined,
```

Comment:
`queue_name` is not directly exposed anymore and would have to be queried for. Could use [reschedule_jobs](https://worker.graphile.org/docs/admin-functions#rescheduling-jobs) instead but this will only work for unlocked jobs.